### PR TITLE
git-checkout: add example to go to previous branch

### PR DESCRIPTION
**Pull request by lmazardo**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/git-checkout.md [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
 `git checkout {{branch_name}}`
 
+- Switch to previous branch:
```
Comment (sbrl) [inline pages/common/git-checkout.md]: Perhaps this will read better?

```suggestion
- Switch to the branch previously checked out:
```

Comment (lmazardo) [reply pages/common/git-checkout.md]: fixed

File: pages/common/git-checkout.md [hunk 2]
```diff
@@ -15,6 +15,10 @@
 
 `git checkout {{branch_name}}`
 
+- Switch to the branch previously checked out:
```
Comment (agnivade) [inline pages/common/git-checkout.md]: Switch to the previously checked out branch ?

In the other examples, "branch" almost always comes at the end.

Comment (lmazardo) [reply pages/common/git-checkout.md]: Fixed.

**Pull request merged**